src/qibolab/dummy.yml

```diff
@@ -107,6 +107,11 @@ native_gates:
             - {type: virtual_z, phase: 0.0, qubit: 2}
             - {duration: 30, amplitude: 0.05, shape: Rectangular(), coupler: 3, relative_start: 0,
                 type: coupler}
+            CNOT:
+            - {duration: 40, amplitude: 0.3, shape: 'Drag(5, -0.02)', frequency: 4150000000.0,
+                relative_start: 0, phase: 0, type: qd, qubit: 2}
+            - {type: virtual_z, phase: 0.0, qubit: 1}
+            - {type: virtual_z, phase: 0.0, qubit: 2}
```

Comment:
Is this supposed to be a realistic representation of CNOT, or just something for now that should be clarified later?

Reply:
Not realistic, everything in the dummy platform is just a placeholder. It will be deserialized during the platform initialization in tests but it is not used in any execution as the dummy platform just returns random numbers.

That being said, the more realistic this is the better but I am not very familiar with CNOT calibration. I believe it requires a drive pulse on the line of one of the qubits with the frequency of the other qubit, which is what I put in this example.

Reply:
Yeah, the reason I asked is that the cross resonance CNOT sequence is more complicated and involves driving both the control and target qubit. The control qubit is driven with both pulses on its own frequency and on the frequency of the other qubit. The pulses of own frequency is ordinary pi-pulse and can be DRAG, while the pulses of other qubit's frequency and the pulses of target qubit are most-probably a flat-topped longer pulses. There is also a simpler version that does not involve pulses on the target qubit.